content/en/storage-provider/how-providing-works/index.md

```diff
@@ -0,0 +1,215 @@
+
+## Hardware requirements
+
+The hardware requirements for providers are very demanding, but once you acquire them, they're not expected to increase in the foreseeable future. Money spent on hardware should provide you with many years of reliable service, paying for itself several times over.
+
+You'll need this hardware, not just to facilitate storage, but also to regularly run some computationally expensive operations, called proofs. You need to regularly run proofs that show that you're continuing to dedicate storage space to the same data you received.
+
+Here's a general idea of the minimum hardware you'll need for providing storage:
+
+- An 8+ core CPU
+- 128 GiB of RAM at the very least
+- A powerful GPU to speed up SNARK computations
+- 1TiB NVMe-based disk space for cache storage
+- Additional hard drives for storing _sealed sectors_, the Lotus chain, and more.
+
+For more detail, see:
+
+- [Hardware requirements](./storage-provider/hardware-requirements/#general-hardware-requirements)
+- [ESPA Module 3C - Hardware requirements](https://www.youtube.com/watch?v=RKihuvJpev4&t=126s)
+
+Every provider's situation is different, but to give you an idea of the level of investment and the possible return, here is one example:
+
+For an 18-month period, an investment of $147k covered combined capital expenses, operating expenses, and FIL loan with 15% collateral. In this example, the storage provider received $267k in FIL rewards (starting at $21), for a profit of $120k over 18 months.
+
+See [Filecoin Storage Provider Bootcamp, by ESPA](https://www.youtube.com/watch?v=T-TgPILQD3c) (at 14:47).
+
+## Install Lotus application
+
+To store files, you'll need to install Lotus.
+
+Lotus is an application that tells computers how to talk to other computers (that follow the same rules), so they can store data for clients.
+
+See [Lotus > Install > Prerequisites](https://lotus.filecoin.io/lotus/install/prerequisites/).
+
+## Storage provider collateral
+
+Most permissionless blockchain networks require an upfront investment in resources to participate in the consensus. The more power an entity has on the network, the greater the share of total resources it needs to own, both in terms of physical resources and/or staked tokens (collateral).
+
+Filecoin includes three different collateral mechanisms:
+
+- Initial pledge collateral (initial commitment)
+- Block reward as collateral (vesting block rewards over time)
+- Storage deal provider collateral (aligns incentives between storage provider and client)
+
+See [Filecoin Spec](https://spec.filecoin.io/#section-systems.filecoin_mining.miner_collaterals).
+
+## Gas fees
+
+You must pay a nominal gas fee for resources used for sealing storage. You can set a gas limit, above which a transaction will not execute, allowing for control.
+
+![Gas fees are negligible compared to collateral.](collateral-gas.png)
```

Comment:
What's the purpose of this screenshot? It doesn't seem relevant to this page. Maybe we should take it out?

Reply:
Helps clarify an aspect of costs. Says, "Don't worry, gas fees are negligible."..... But I agree, would be better to have something more relevant, like an example of cost of hardware. Point well taken. We can remove.